rockcraft.yaml

```diff
@@ -87,9 +89,51 @@ parts:
     stage:
       - opt/spark/jars
 
-  user-setup:
+  client-scripts:
     plugin: nil
     after: [hadoop-jars]
+    build-packages:
+      - wget
```

Comment:
Why `wget` for `client-scripts` and `curl` for `kubectl-bin`?

Reply:
fixed. everything uses wget now.